acf-json/group_6089d6189b18e.json

```diff
@@ -0,0 +1,103 @@
+{
+    "key": "group_6089d6189b18e",
+    "title": "Author information",
+    "fields": [
+        {
+            "key": "field_6089d62743512",
+            "label": "Name",
+            "name": "uds_news_author_name",
+            "type": "text",
+            "instructions": "",
+            "required": 0,
+            "conditional_logic": 0,
+            "wrapper": {
+                "width": "",
+                "class": "",
+                "id": ""
+            },
+            "default_value": "",
+            "placeholder": "",
+            "prepend": "",
+            "append": "",
+            "maxlength": ""
+        },
+        {
+            "key": "field_6089d71d4de0f",
+            "label": "Title",
+            "name": "uds_news_author_title",
+            "type": "text",
+            "instructions": "",
+            "required": 0,
+            "conditional_logic": 0,
+            "wrapper": {
+                "width": "",
+                "class": "",
+                "id": ""
+            },
+            "default_value": "",
+            "placeholder": "",
+            "prepend": "",
+            "append": "",
+            "maxlength": ""
+        },
+        {
+            "key": "field_6089d66b43513",
+            "label": "Email",
+            "name": "uds_news_author_email",
+            "type": "email",
+            "instructions": "",
+            "required": 0,
+            "conditional_logic": 0,
+            "wrapper": {
+                "width": "",
+                "class": "",
+                "id": ""
+            },
+            "default_value": "",
+            "placeholder": "",
+            "prepend": "",
+            "append": ""
+        },
+        {
+            "key": "field_6089d67c43514",
+            "label": "Phone",
+            "name": "uds_news_author_phone",
+            "type": "number",
+            "instructions": "",
```

Comment:
Because this a Number field, the phone number must be entered as digits only (and correct formatting is then performed by the client). This field needs instructions explaining this. I was confused when first entering info. The field does not prevent typing the number as a string (including dashes), causing the field to not be saved.